src/devices/nm-acd-manager.c

```diff
@@ -117,21 +117,15 @@ static const char *
 _acd_error_to_string (int error)
 {
 	if (error < 0)
-		return strerror(-error);
+		return g_strerror (-error);
 
 	switch (error) {
 	case _N_ACD_E_SUCCESS:
 		return "success";
-	case N_ACD_E_DONE:
-		return "no more events (engine running)";
-	case N_ACD_E_STOPPED:
-		return "no more events (engine stopped)";
 	case N_ACD_E_PREEMPTED:
 		return "preempted";
 	case N_ACD_E_INVALID_ARGUMENT:
 		return "invalid argument";
-	case N_ACD_E_BUSY:
-		return "busy";
 	}
 	return NULL;
```

Comment:
maybe `g_return_val_if_reached (NULL)`. In this case, n-acd is a static library that is compiled in. This isn't (really) an external component, and an unknown error code *is* a bug that we should assert against. And then drop the `acd_error_to_string()` macro.

Reply:
Okay.